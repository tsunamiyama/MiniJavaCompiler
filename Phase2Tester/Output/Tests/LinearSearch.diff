--- SelfTestCases/LinearSearch.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Output/Tests/LinearSearch.out	2021-02-13 18:30:51.568603900 -0800
@@ -1,15 +1,5 @@
-10
-11
-12
-13
-14
-15
-16
-17
-18
-9999
-0
-1
-1
-0
-55
+--- Error (ProgramFault) ---
+use of uinitialized value
+  Main() line 18
+  LS.Start(<Heap.0>+0 10) line 32
+  LS.Init(<Heap.0>+0 10) line 271
