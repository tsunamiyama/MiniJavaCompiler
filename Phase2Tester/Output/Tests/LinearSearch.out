--- Error (ProgramFault) ---
use of uinitialized value
  Main() line 18
  LS.Start(<Heap.0>+0 10) line 32
  LS.Init(<Heap.0>+0 10) line 271
